diff --git a/node_modules/@azure/msal-browser/dist/network/FetchClient.mjs b/node_modules/@azure/msal-browser/dist/network/FetchClient.mjs
index bcc88be..9595e09 100644
--- a/node_modules/@azure/msal-browser/dist/network/FetchClient.mjs
+++ b/node_modules/@azure/msal-browser/dist/network/FetchClient.mjs
@@ -61,23 +61,25 @@ class FetchClient {
         let responseStatus = 0;
         let responseHeaders = {};
         try {
-            response = await fetch(url, {
+            response = await requestUrl({
+				url,
                 method: HTTP_REQUEST_TYPE.POST,
                 headers: reqHeaders,
                 body: reqBody,
             });
         }
         catch (e) {
+			console.error(e)
             throw createBrowserAuthError(window.navigator.onLine
                 ? postRequestFailed
                 : noNetworkConnectivity);
         }
-        responseHeaders = getHeaderDict(response.headers);
+        responseHeaders = response.headers;
         try {
             responseStatus = response.status;
             return {
                 headers: responseHeaders,
-                body: (await response.json()),
+                body: response.json,
                 status: responseStatus,
             };
         }
